index.html

```diff
@@ -1 +1,3 @@
-<p>Hello world</p>
\ No newline at end of file
+<p>Hello world</p> <br>
```

Comment:
@satyanivas02 I am not able to view the previous 3 commits which you squashed .
can you do this in this way:

1. do 3 commits in local
2. push that 3 commits in github
3. squash 3 commits into single commit
4. push this final commit